src/ModelXSD/StartupServicesModel.xsd

```diff
@@ -6,6 +6,11 @@
     <xs:documentation>Microsoft Azure Service Fabric StartupServices Model. Version: V1_RTM_3 [MajorVersion_Milestone_ChangeNo]</xs:documentation>
   </xs:annotation>
   <xs:element name="StartupServicesManifest" type="StartupServicesManifestType"/>
+  <xs:element name="StartupServices" type="StartupServiceInstanceDefinitionType">
```

Comment:
To prevent further iterations, can you confirm if this is "StartupServices" or just "Services" ?

Reply:
it is StartupServices